apps/framework-cli/src/cli/routines/peek.rs

```diff
@@ -89,6 +89,8 @@ pub async fn peek(
 
         consumer_ref = create_consumer(&project.redpanda_config, &[("group.id", &group_id)]);
         let consumer = &consumer_ref;
+        // in dmv2 we still have version as the suffix for topics
+        // but not so for tables. we might want to change that
```

Comment:
I believe we are adding back versions in tables for backward compatibility with existing deployments